### ISSUE TITLE
Approve Added Funds

### ISSUE THREAD

**Issue by shannondwray** [opencollective/opencollective#6861]
## Who is your user?
A user who "contributes" towards a collective via added funds. Which is triggered by a 'fiscal host' 

## What are they trying to achieve?
Add an approved added funds feature where the user needs to confirm that they want to be referenced to have added the funds to the collective. 

## How are they currently doing this?
No approval needs to happen so spam fiscal hosts can add funds from random users. This has meant that I've received complaints and rejected added funds. 

## How well understood is this problem?

 - [ ] Surfacing: if you feel this is a seed of an idea that needs to be researched.
 - [ ] Understanding: if you've done some research and feel it is already well understood and ready for shaping.
 - [ ] Shaping:  if you feel a solution is clearly defined and can be made into a pitch ([read more about shaping](https://basecamp.com/shapeup/1.1-chapter-02))
 - [ ] Pitching: if you feel that it is ready to be presented for prioritisation ([read more about pitching](https://basecamp.com/shapeup/1.5-chapter-06))

**Comment by Betree** [comment]
I agree that random admins shouldn't be allowed to add funds to any account. The rule should be:
- It's a trusted or internal host: add funds directly
- Otherwise, either disable added funds (`Contact us to enable this feature`), limit it to administrated account, or add a confirmation like described above

@shannondwray Could you link some of the SPAM you've noticed for added funds (here or in PM) to see what we're trying to protect from?

**Comment by SudharakaP** [comment]
> I agree that random admins shouldn't be allowed to add funds to any account. The rule should be:
> 
>     * It's a trusted or internal host: add funds directly
> 
>     * Otherwise, either disable added funds (`Contact us to enable this feature`), limit it to administrated account, or add a confirmation like described above
> 
> 
> @shannondwray Could you link some of the SPAM you've noticed for added funds (here or in PM) to see what we're trying to protect from?

This one I do get from time to time. Recently there was a user who was doing this heavily and other users were complaining on the support channel about these random added funds created by the user. Therefore I strongly advocate the idea of enabling it only to trusted or internal hosts or enabling it on demand. If you want more examples of added fund spamming let me know and I can search for those tickets and provide some examples of this.
